packages/widget-v2/src/pages/SwapPage/SwapPage.tsx

```diff
@@ -296,46 +302,48 @@ export const SwapPage = () => {
                   paddingRight: 13,
                 }}
               >
-                {formattedBalance && (
-                  <>
-                    <TransparentButton
-                      onClick={() => {
-                        connectedWalletModal.show();
-                      }}
-                      style={{
-                        padding: "8px 13px",
-                        alignItems: "center",
-                        gap: 8,
-                      }}
-                    >
-                      {sourceAccount && (
-                        <img
-                          style={{ objectFit: "cover" }}
-                          src={sourceAccount?.wallet.logo}
-                          height={16}
-                          width={16}
-                        />
-                      )}
-                      {formattedBalance}
-                    </TransparentButton>
+                {
+                  formattedBalance && (
+                    <>
+                      <TransparentButton
+                        onClick={() => {
+                          connectedWalletModal.show();
+                        }}
+                        style={{
+                          padding: "8px 13px",
+                          alignItems: "center",
+                          gap: 8,
+                        }}
+                      >
+                        {sourceAccount && (
+                          <img
+                            style={{ objectFit: "cover" }}
+                            src={sourceAccount?.wallet.logo}
+                            height={16}
+                            width={16}
+                          />
+                        )}
+                        {formattedBalance}
+                      </TransparentButton>
 
-                    <TransparentButton
-                      disabled={!sourceBalance || sourceBalance?.amount === "0"}
-                      onClick={handleMaxButton}
-                      style={{
-                        padding: "8px 13px",
-                        alignItems: "center",
-                      }}
-                    >
-                      Max
-                    </TransparentButton>
-                  </>
-                )}
-              </Row>
+                      <TransparentButton
+                        disabled={!sourceBalance || sourceBalance?.amount === "0"}
+                        onClick={handleMaxButton}
+                        style={{
+                          padding: "8px 13px",
+                          alignItems: "center",
+                        }}
+                      >
+                        Max
+                      </TransparentButton>
+                    </>
+                  )
+                }
+              </Row >
             )
           }
         />
-        <Column align="center">
+        < Column align="center" >
```

Comment:
nit: what is this linter? lol why is it adding spaces like that

Reply:
xd